build/makewheel.py

```diff
@@ -0,0 +1,33 @@
+from waflib.Build import BuildContext
+from waflib import Errors
+
+import glob
+import os
+import shutil
+import subprocess
+
+
+class makewheel(BuildContext):
+    '''builds a wheel for easy installation'''
+    cmd='makewheel'
+    fun='build'
+
+    def execute(self):
+        self.restore()
+        if not self.all_envs:
+            self.load_envs()
+
+        if not self.env['SETUP_PY_DIR']:
+            raise Errors.WafError('Could not make wheel. '
+                'Unable to find setup.py')
+        shutil.copyfile(os.path.join(self.env['SETUP_PY_DIR'],
+            'setup.py'), 'setup.py')
+        self.to_log('Creating wheel\n')
+        subprocess.call(['pip', 'wheel', '.', '--wheel-dir', '.', '--no-deps'])
+        os.remove('setup.py')
+
+        wheels = glob.glob('*.whl')
+        for wheel in wheels:
```

Comment:
Do you have to do the move like this?  Can't you just set `--wheel-dir` above to `install_bindir`?  Or am I misunderstanding what that flag does?